src/lifecycle/attached.js

```diff
@@ -16,13 +16,11 @@ export default function (opts) {
     var info = data(this, opts.id);
     var isNative = this.attachedCallback;
 
-    if (info.attached) {
-      return;
-    }
-
+    if (info.attached) return;
     info.attached = true;
+    info.detached = false;
```

Comment:
Because it doesn't really matter if it's before or after the call and it reads better to be set all in one spot.